src/sentry/sentry_metrics/indexer/base.py

```diff
@@ -51,6 +54,19 @@ def from_string(cls: Type[KR], key: str, id: int) -> KR:
         return cls(int(org_id), string, id)
 
 
+@dataclass(frozen=True)
+class UseCaseResult:
```

Comment:
Could we rename this to `UseCaseKeyResult` and the others likewise?

Reply:
Done